### ISSUE TITLE
Story setting, descriptor for climate

### ISSUE THREAD

**Issue by cjcodeproj** [cjcodeproj/vtmedia-schema#114]
The in progress representation for a story setting so far is this.

```
 <settings>
   <primary>
      <where>
         <place><ci>Las Vegas</ci><st abbrv='NV'>Nevada</st></place>
       </where>
   </primary>
 </settings>
```

The `<place>` element (or the `fictionalPlace` element)  is the most descriptive/detail element available to describe a place.

But there should be a couple of optional elements to describe aspects of the setting, including climate, or geographical density.

Possible descriptors include `desert`, `forest`, `tundra`.

Once the initial setting is merged into development, there should be an investigation into sub elements that can enhance the description.

References: https://github.com/cjcodeproj/vtmedia-schema/issues/79
